test/hotspot/jtreg/vmTestbase/nsk/share/jdb/Launcher.java

```diff
@@ -32,7 +32,7 @@
 
 /**
  * This class provides launching of <code>jdb</code> and debuggee in local
- * or remote mode according to test command line options.
+ * mode according to test command line options.
```

Comment:
As now there is only local mode, I think there is no sense to mention it at all:
```
 * This class provides launching of <code>jdb</code> and debuggee
 * according to test command line options.
```